apps/healthlocker/priv/repo/migrations/20180122163011_create_school_feedback.exs

```diff
@@ -0,0 +1,49 @@
+defmodule Healthlocker.Repo.Migrations.CreateSchoolFeedback do
+  use Ecto.Migration
+
+  def change do
+    create table(:school_feedbacks) do
```

Comment:
@RobStallion could we "_normalize_" this schema/table to reduce the repetition?

Reply:
@nelsonic Looking into alternatives now. 👍